### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by SayantanKhanra10**
Added the feature to add an user_group as tenant operator of the company.

### CODE REVIEW

**Comment by Vairavaprakash** [comment]
Hi Sayantan Khanra,

add_user_group_as_operator is designed for accepting only one user group as input, can we add support for accepting a list of user group

**Comment by SayantanKhanra10** [comment]
Hi @Vairavaprakash ,

Added that in the latest commit.

File: cvpysdk/organization.py [hunk 1]
```diff
@@ -963,6 +961,38 @@ def tenant_operator(self):
         tenant_operators = self._organization_info.get('organizationProperties', {}).get('operators', [])
         return [role['user']['userName'] for role in tenant_operators]
 
+    def add_user_groups_as_operator(self, user_group_list, request_type):
+        """Update the local user_group as tenant operator of the company
+
+        Args:
+            user_group_list		(Str)  -- user group list
```
Comment (Vairavaprakash) [inline cvpysdk/organization.py]: can you please change accepted type as list

Comment (SayantanKhanra10) [reply cvpysdk/organization.py]: Done!

**Pull request merged**